content/productivity/custom-distribute-silver/plugin.js

```diff
@@ -0,0 +1,444 @@
+// Custom distribute Silver
```

Comment:
Can you say why its different from the other silver plugin here. Also state the constants and defaults that people can change. " By default sends a min of 2000 silver to a concurrent 5 ... all configurable by changing the constants at the top of the file"

Reply:
Also the timer " by default runs ever 30 seconds" etc

Reply:
Also please explain how the plugin works and the pair list. I believe the plugin works not automatically, but rather by you selecting a single asteroid and then selecting who it should send stuff to. This is called the pair list? Which can be 'backed up' by downloading pair list, and then 'uploading' it when you restart the browser and the plugin?

Reply:
I have add comment at the top of the file.
Thanks for your review.